src/lib/tls/tls_policy.cpp

```diff
@@ -563,6 +619,14 @@ std::vector<std::string> Strict_Policy::allowed_key_exchange_methods() const
    }
 
 bool Strict_Policy::allow_tls12()  const { return true;  }
+bool Strict_Policy::allow_tls13()  const
+{
+#if defined(BOTAN_HAS_TLS_13)
+   return true;
+#else
+   return false;
+#endif
```

Comment:
Should this just inherit the default Policy impl instead?

Reply:
Sounds reasonable. At least until "strict" doesn't mean to prohibit certain TLS versions. I'm removing all three "allows"...